src/entry_window.py

```diff
@@ -204,5 +207,36 @@ def open_existing_project(self):
         # add existing project to recent projects
         self.add_recent_project(selected_project)
 
+        self.сheck_project_version(selected_project)
+
         # emit signal with path to project file
         self.open_project_signal.emit(selected_project)
+
+    def сheck_project_version(self, project_path):
+        settings_filename = "settings.yaml"
+        project_settings_filename = str(pathlib.Path(project_path, settings_filename))
```

Comment:
I think we have `project_settings_filename` somewhere in `PathBuilder.<...>` better not to invent something here